usaspending_api/common/recipient_lookups.py

```diff
@@ -0,0 +1,59 @@
+from usaspending_api.recipient.models import RecipientLookup
+
+
+def obtain_recipient_uri(recipient_name, recipient_unique_id, parent_recipient_unique_id=None):
+    if not recipient_unique_id:
+        return create_recipient_uri_without_duns(recipient_name, recipient_unique_id, parent_recipient_unique_id)
+
+    return fetch_recipient_hash_using_name_and_duns(recipient_name, recipient_unique_id, parent_recipient_unique_id)
+
+
+def create_recipient_uri_without_duns(recipient_name, recipient_unique_id, parent_recipient_unique_id=None):
+    recipient_hash = generate_missing_recipient_hash(recipient_name, recipient_unique_id)
```

Comment:
`generate_missing_recipient_hash` parameters appear to be out of order from the actual function signature.

Reply:
Good find